uint16[] id_list

float64[] position_deg
float64[] rotation